### DISCUSSION TITLE
Open Tab twice if wanted

### DISCUSSION THREAD

**Discussion by Friedolito** [oxdc/obsidian-vertical-tabs#92] [Ideas]
Hey thank you for working on vertical tabs! 

I love the feature to prevent opening the same note in multiple tabs.
But I noticed that from time to time I actually need it (usually when working on a larger file and I want to work at two places at once). 
I would wish for a command to "duplicate" tab and this will escape the automatic closure. 
Right now I have to disable / reenable the feature in the settings. 

Best Regards :)

**Comment by oxdc** [comment]
Hi @Friedolito ! Thank you for sharing your use case. I think Tab Navigation should be set once for all. Constantly switching between navigation modes or manually deduplicating tabs by executing commands, both are unacceptable for me. It's a waste of energy.

I'd like to include an option that allows Vertical Tabs to ignore duplicated tabs in additional windows. When you need multiple tabs of a lengthy note, you can:
- Open the note;
- Right click the tab;
- Click "Open in new window".

How does this solution sound to you?

**Reply by churnish** [reply]
I have the same use case as OP. To me, the solution suggested would be far from ideal because I don't like to split my Obsidian instance into multiple windows. In part, because [a pop-up window is not a proper Obsidian instance](https://forum.obsidian.md/t/option-for-pop-out-windows-to-have-panes-as-main-window-like-file-explorer-and-outline/41963).

Though the suggested solution would still be a welcome stopgap.

Also, @Friedolito you might want to take a look at [Mononote](https://obsidian.md/plugins?id=mononote) as an alternative solution—it dedupes tabs only within the same tab group.

**Reply by oxdc** [reply]
> dedupes tabs only within the same tab group.

Hi @greetclammy ! Thank you. I'm working on this option, will be available in the next release.

**Comment by oxdc** [comment]
Implemented functions:

- Automatically reveal the active note after deduplication: "Unhide Group A and bring up Note 1 that's already open."
- New option: "only deduplicate tabs within same group."
- New option: "ignore duplicated tabs in additional windows."
- New command: manually duplicating all tabs.

Please let me know if I missed something. Thank you!
